keps/prod-readiness/sig-apps/5547.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 5547
+alpha:
+  approver: "@soltysh"
```

Comment:
Based on [PRR dashboard](https://github.com/orgs/kubernetes/projects/229/views/2) this will be done by @johnbelamaric . Ideally we should avoid having  SIG approver and PRR approver be the same person. So please change accordingly.